### DISCUSSION TITLE
ReactionFromSmarts should have an option to change parser settings

### DISCUSSION THREAD

**Discussion by ruslankotl** [rdkit/rdkit#4903] [Ideas] [answered]
I am using RDKit to filter out C-H activation reactions from a database and have noticed there is a way to alter `Chem.SmartsParserParams().mergeHs` from `True` to `False` to test if that helps keeping track of hydrogens in the reactions. However, the parameters can only be altered for `MolFromSmarts` as C++ implementation of `ReactionFromSmarts` has no room for custom parser settings.

Would it be possible to enable parser parameter change for reactions as well?

**Comment by greglandrum** [comment, marked as answer]
Definitely. I've created an enhancement request for it here: #4905
